assets/js/auto-merge.js

```diff
@@ -0,0 +1,66 @@
+(function($, undefined) {
+    "use strict";
+	
+	$(document).ready(function () {
+		
+		/**
+		 * Sets up the merge interface (using mergely). 
+		 */
+		function mergelySetUp(div, left, right)
+		{
+			div.mergely({
+				width: "auto",
+				height: 400,
+				sidebar: false,
+				cmsettings: { readOnly: false, lineNumbers: true, lineWrapping: true },
+				lhs: function(setValue) {
+					setValue(left.html());
+				},
+				rhs: function(setValue) {
+					setValue(right.html());
+				}
+			});
+		}
+
+		mergelySetUp($(".compare-introduction"),$("#your_introduction"),$("#id_introduction"));
+		mergelySetUp($(".compare-conclusion"),$("#your_conclusion"),$("#id_conclusion"));
+
+		$("#compare-editor-lhs").append("Votre Version");
+		$("#compare-editor-rhs").append("La version courante");
+		
+		
+		/**
+	     * Merge content
+	     */
+	    $(".merge-btn").on("click", function(e){
+
+	    	e.stopPropagation();
+	        e.preventDefault();
+	        
+	        var button = $(this);
+	        var classList = button.attr("class").split(/\s+/);
```

Comment:
hum pas sûr que ça soit la méthode la plus facile d'autant qu'il existe déjà l'attribut classList (`this.classList`).
Ensuite plutôt qu'un simple for, je suis d'avis d'utiliser un `this.classList.forEach(function(){...}`/